unitTests/ui/CategoryFacet/CategoryFacetTest.ts

```diff
@@ -27,26 +27,6 @@ export function CategoryFacetTest() {
     let test: IBasicComponentSetup<CategoryFacet>;
     let simulateQueryData: ISimulateQueryData;
 
-    describe('exposes options', () => {
```

Comment:
this test was removed since I had difficulties recreating it with the new setup and after talking with @olamothe, since this test only checks if a private function got called and the result of the function is already tested in the dependsOnManagerTest, this test was not necessary